src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -619,13 +648,36 @@ private static Device makeDevice(Device device) {
                 .build();
     }
 
-    private static Site makeSite(Site site, ExtImpRubicon rubiconImpExt) {
-        return site == null ? null : site.toBuilder()
+    private static Site makeSite(Site site, String impLanguage, ExtImpRubicon rubiconImpExt) {
+        return site == null
```

Comment:
It make sense to check `impLanguage` for null (or empty?), otherwise empty `Content` object will be created.